### PR TITLE
[PRM] Bugs and Improvements requested by Channel Team

### PR DESCRIPTION
**Pull request by Elvinho**

### CODE REVIEW

**Comment by Elvinho** [comment]
ci:test:sf

**Comment by Manda-vit** [comment]
ci:test:stable

**Comment by Manda-vit** [comment]
ci:test:relevant

**Comment by Elvinho** [comment]
ci:test:stable

**Comment by Elvinho** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/button-welcome-banner/index.js [hunk 1]
```diff
@@ -6,11 +6,16 @@
...
+	'?random=' + new Date().getTime() + Math.floor(Math.random() * 1000000);
+
 if (layoutMode !== 'edit') {
 	for (const videoButton of videoButtons) {
 		videoButton.onclick = () =>
 			Liferay.Util.openModal({
-				bodyHTML: `<iframe width="100%" height="500" src=${configuration.videoButtonLink} frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>`,
+				bodyHTML: `<iframe width="100%" height="500" src=${
+					configuration.videoButtonLink + randomTime
+				} loading="lazy" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>`,
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/button-welcome-banner/index.js:18]:
```suggestion
				bodyHTML: `<iframe width="100%" height="500" src=${
					configuration.videoButtonLink
				}?random=${randomTime} loading="lazy" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>`,
```

**Pull request closed without merging**